### ISSUE TITLE
Limiting bits is not working

### ISSUE THREAD

**Issue by sudo-su-FDEL** [bitbrute/evillimiter#109]
Seems as if the limiting kbits is now working anymore, tried it on my phone and it can still surf the web and play youtube videos even though its been "limited".
Also with the blocking option, it blocks SOME of the applications that run on my phone, but when i go to youtube app, it plays youtube videos as if nothing was blocking it.

I wonder if anyone else has had this issue.

**Comment by imneonizer** [comment]
This works for me

````
sudo evillimiter --flush
````

**Comment by TollyThaWally** [comment]
If some things are being blocked but others aren't, it's likely the services that aren't being blocked are using IPv6, which is not supported. (#6)

**Comment by sudo-su-FDEL** [comment]
I don't think that might be the issue, hence devices my household (from what im aware of) has not been configured to use IPV6, not unless companies came out with some new update that forces devices to use IPV6 or some protocol that has to do with IPV6 or something that I dont know of, I think it might be something else

**Comment by TollyThaWally** [comment]
A lot of routers and devices have IPv6 enabled by default now, you can check if your devices are using it by visiting sites such as https://test-ipv6.com/
